src/index.html

```diff
@@ -5,8 +5,293 @@
+            src="images/imac.jpeg"
+            alt="Product"
+          >
+          <h3 class="product__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+          <p class="product__code">Код товара: 195434</p>
+          <div class="product__rating">
+            <div class="product__stars stars stars--4">
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+            </div>
+            <p class="product__review">Отзывов: 5</p>
+          </div>
+          <div class="product__price price">
+            <p class="price__text">Цена</p>
+            <p class="price__value">69 999 грн</p>
+          </div>
+          <a
+            class="product__buy"
+            href=""
+          >
+            Купить
+          </a>
+        </div>
+
+        <div class="catalog__product product">
+          <img
+            class="product__photo"
+            src="images/imac.jpeg"
+            alt="Product"
+          >
+          <h3 class="product__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+          <p class="product__code">Код товара: 195434</p>
+          <div class="product__rating">
+            <div class="product__stars stars stars--4">
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+            </div>
+            <p class="product__review">Отзывов: 5</p>
+          </div>
+          <div class="product__price price">
+            <p class="price__text">Цена</p>
+            <p class="price__value">69 999 грн</p>
+          </div>
+          <a
+            class="product__buy"
+            href=""
+          >
+            Купить
+          </a>
+        </div>
+
+        <div class="catalog__product product">
+          <img
+            class="product__photo"
+            src="images/imac.jpeg"
+            alt="Product"
+          >
+          <h3 class="product__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+          <p class="product__code">Код товара: 195434</p>
+          <div class="product__rating">
+            <div class="product__stars stars stars--4">
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+            </div>
+            <p class="product__review">Отзывов: 5</p>
+          </div>
+          <div class="product__price price">
+            <p class="price__text">Цена</p>
+            <p class="price__value">69 999 грн</p>
+          </div>
+          <a
+            class="product__buy"
+            href=""
```

Comment:
Не залишай пустим href, інакше сторінка при каждому кліку буде перезагружатись. Задай як дефолтне значення "#"
```suggestion
            href="#"
```